src/endf/_records.cpp

```diff
@@ -25,11 +25,12 @@ double cfloat_endf(const char* buffer)
   int n = std::strlen(buffer);
 
   int i;
+
   for (i = 0; i < n; ++i) {
     char c = buffer[i];
 
     // Skip whitespace characters
-    if (c == ' ') continue;
+    if (std::isspace(c)) continue;
```

Comment:
The ENDF format actually does not allow characters like `\t` and `\f` which are handled by `std::isspace`. Sticking with the original check against a space should be sufficient.